CHANGELOG.md

```diff
@@ -83,6 +83,7 @@ The table below shows which release corresponds to each branch, and what date th
 - [#2376][2376] Return buffered data on first EOF in tube.readline()
 - [#2387][2387] Convert apport_corefile() output from bytes-like object to string
 - [#2388][2388] libcdb: add `offline_only` to `search_by_symbol_offsets`
+- [#2413][2413] libcdb: improve the search speed of `search_by_symbol_offsets`
```

Comment:
rebase on latest dev please and move this to the 4.15.0 changelog

Reply:
I don't know how to rebase just the `CHANGELOG.md`. Do I need open a new PR?